docs/installation.md

```diff
@@ -218,19 +218,234 @@ On Linux, as an optional post-installation task, you may wish to setup the bot t
+
+```bash
+conda create -n freqtrade-jupyter-conda python=3.8 pandas numpy ta-lib git wheel virtualenv jupyter
+```
+
+the same works for [docker](docker.md), for spyder and other useful programs.
+
+Further read on the topic:
+
+https://towardsdatascience.com/a-guide-to-conda-environments-bc6180fc533?gi=1db972389cd1
+
+https://metager.de/meta/meta.ger3?eingabe=ardsdatascience+guide+to+conda+environment
+
+!!! NOTE:
+It may happen that creating a new Conda environment, populated with selected packages at the moment of creation, takes less time than installing a large, heavy dependent package into the old environment,
+takes less time than installing a large, heavy dependent package into the old environment.
+
+##### Enter/exit freqtrade-conda venv:
+
+to enter choosen conda environment
+
+```bash
+conda activate freqtrade-conda
+
+```
+
+to exit conda environment
+
+```bash
+conda deactivate						
+```
+
+#### 4. Freqtrade Instalation
+
+Within the newly created freqtrade-conda environment, download and install Freqtrade. To process --install command, may take a few minutes
+
+```bash
+# download freqtrade
+git clone https://github.com/freqtrade/freqtrade.git
+
+# enter downloaded directory 'freqtrade'
+cd freqtrade      
+
+# run setup
+./setup.sh --install                                        
+```
+
+Create virtual environment (yes, python3-venv environment, inside conda-environment), where Freqtrade program can run.
+
+Running python3-venv seems to be requirement of the Freqtrade program itself.
+
+```bash
+# create venv environment, inside hidden directory /freqtrede/.env
+python3 -m venv ./.env/
+
+# run the python3-venv environment
+source ./.env/bin/activate
+
+# install last required package            
+pip install -e .						
+```
+
+##### pip install within conda, a reminder:
+
+The documentation of conda says that pip should NOT be used within conda, because internal problems can occur.
+However, they are rare
+
+That is, why the conda-forge channel is preferred: (and if no library is available in conda, you have to use pip)
+
+* more libraries are available (less need for pip)
+* conda-forge works better with pip
+* the libraries are newer
+
+
+#### 5. You are ready
+
+You are ready to run, create the user directory and configuration file, run the program dry to verify that everything is working, and run a backtest to double check.
+
+```bash
+freqtrade create-userdir --userdir user_data
+freqtrade new-config --config config.json
+freqtrade download-data -t 5m 15m 1h --days 100
+freqtrade trade --strategy SampleStrategy  
+freqtrade backtesting -s SampleStrategy
+```
+
+important shortcuts
+
+```bash
+# activate base environment
+conda activate
+
+# activate freqtrade-conda environment
+conda activate freqtrade-conda
+
+#deactivate any conda environments
+conda deactivate
+
+# list installed conda environments
+conda env list
+
+# activation/deactivate the venv
+source ./.env/bin/activate
```

Comment:
I think nesting a virtual env within a conda installation is screaming loud for problems. See above as to why this is not necessary.

Reply:
Nesting won't make problems, so far as you run run them in sequence.

At the same time, i found that freqtrade, may have some problems, if one not run it in python3-vevn. 
I would keep it based on my experience. But lets keep this open.

In whatever way, if you will confirm to delete it, i would check it again, and let you know what is the outcome.

Reply:
that's the point - running them in sequence (and also not breaking the sequence later by installing something with conda, which may or may not be present in the virtual environment already).

It's easy if you're used to working with conda - can be difficult for a user not used to working with conda - as you google "how to install X with conda" - and the first command that pops up is `conda install X` (maybe with --channel, depending on the dependency ....).

Reply:
Like i really think that, the only way to cover it, is to be very clear in Documentation, there is no other (and simple) way. What is alternative - writing protection plugins, that will defend those without possibility to read?